src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -132,195 +132,109 @@ private function getPathOperation(string $operationName, array $operation, strin
-
-                    if (!isset($pathOperation['parameters'])) {
-                        $pathOperation['parameters'] = [
-                            [
-                                'name' => 'id',
-                                'in' => 'path',
-                                'required' => true,
-                                'type' => 'integer',
-                            ],
-                        ];
-                    }
-
-                    if (!isset($pathOperation['responses'])) {
-                        $pathOperation['responses'] = [
-                            '200' => [
-                                'description' => sprintf('%s resource response', $resourceShortName),
-                                'schema' => [
-                                    '$ref' => sprintf('#/definitions/%s', $resourceShortName),
-                                ],
-                            ],
-                            '404' => [
-                                'description' => 'Resource not found',
-                            ],
-                        ];
-                    }
-                }
-                break;
-
-            case 'POST':
-                if (!isset($pathOperation['consumes'])) {
-                    $pathOperation['consumes'] = $mimeTypes;
-                }
-                if (!isset($pathOperation['produces'])) {
-                    $pathOperation['produces'] = $mimeTypes;
-                }
-
-                if (!isset($pathOperation['summary'])) {
-                    $pathOperation['summary'] = sprintf('Creates a %s resource.', $resourceShortName);
-                }
-
-                if (!isset($pathOperation['parameters'])) {
-                    $pathOperation['parameters'] = [
-                        [
-                            'name' => lcfirst($resourceShortName),
-                            'in' => 'body',
-                            'description' => sprintf('The new %s resource', $resourceShortName),
-                            'schema' => [
-                                '$ref' => sprintf('#/definitions/%s', $resourceShortName),
-                            ],
-                        ],
-                    ];
-                }
-
-                if (!isset($pathOperation['responses'])) {
-                    $pathOperation['responses'] = [
-                        '201' => [
-                            'description' => sprintf('%s resource created', $resourceShortName),
-                            'schema' => [
-                                '$ref' => sprintf('#/definitions/%s', $resourceShortName),
-                            ],
-                        ],
-                        '400' => [
-                            'description' => 'Invalid input',
-                        ],
-                        '404' => [
-                            'description' => 'Resource not found',
+        $pathOperation['tags'] ?? $pathOperation['tags'] = [$resourceShortName];
```

Comment:
IMO that's a bad coding style. Too cryptic and most likely doesn't even have any practical benefit.

It's short-circuit on steroids... lol